sql/core/src/test/scala/org/apache/spark/sql/parquet/ParquetIOSuite.scala

```diff
@@ -107,7 +107,7 @@ class ParquetIOSuiteBase extends QueryTest with ParquetTest {
         // Parquet doesn't allow column names with spaces, have to add an alias here
         .select($"_1" cast decimal as "dec")
 
-    for ((precision, scale) <- Seq((5, 2), (1, 0), (1, 1), (18, 10), (18, 17))) {
+    for ((precision, scale) <- Seq((5, 2), (1, 0), (1, 1), (18, 10), (18, 17), (19, 0), (38, 37))) {
```

Comment:
Ah, yes, you removed the < 8 check too. But shouldn't followParquetFormatSpec = false generate compatible files?

I'm getting a compiler assertion on test compile and I tried cleaning :-S Anyway, must be some sort of local ******.